tests/lambda_tests.py

```diff
@@ -37,6 +38,16 @@ def setUp(self):
         os.environ['TYPE'] = "vpcflow"
         self._logzioUrl = "{0}/?token={1}&type={2}".format(os.environ['URL'], os.environ['TOKEN'], os.environ['TYPE'])
 
+    def tearDown(self):
```

Comment:
if for any reason you won't be able to delete this and the key exists, can it mess with other tests here?